cmd/snap/cmd_snap_op_test.go

```diff
@@ -1302,6 +1292,22 @@ func (s *SnapOpSuite) TestRefreshOneRebooting(c *check.C) {
 
 }
 
+func (s *SnapOpSuite) TestRefreshOneChanDeprecated(c *check.C) {
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, `{"type": "error", "result": {"message": "snap not found", "value": "foo", "kind": "snap-not-found"}, "status-code": 404}`)
```

Comment:
we should check the channel we get here from the client, no?

Reply:
no, this'll print `latest/stable` instead of `stable` for `/stable`; I'm fixing that now.

Reply:
done